monitoring/grafana/README.md

```diff
@@ -4,6 +4,18 @@ that will help you get started quickly introducing monitoring capabilities to yo
 
 Quick Start
 -----------
+Before generating the dashboards on your own, note that you can find the sample
+dashboard in importable JSON format inside the folder
+`grr/monitoring/grafana/grr_grafanalib_dashboards/dashboards_for_use`. From
+there you may import the sample dashboards without re-generating them to your
+Grafana instance. Only if you wish to generate the dashboards, proceed.

```

Comment:
I'd make "Only if you wish to generate the dashboards, proceed." more clear: "Follow the instructions below if you want to rebuild or customise the dashboards".